website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -68,6 +68,10 @@ Your Consul configuration must meet the following requirements to use admin part
 
 - Consul 1.11.1 and newer
 
+### General Networking Requirements
```

Comment:
I image that the next question from users is going to be 'well, how do I do that?' This requirement is fulfilled by configuring firewalls in whatever infrastructure environment, correct? Not Consul-specific settings? Should we add something like "Refer to the documentation for your infrastructure environment for instructions on how to configure routes between nodes." Or am I misunderstanding?

Reply:
I expanded, we actually do want to make sure clients also can communicate to servers via RPC, Gossip and HTPS as well for general networking requirements.